tuyaha/devices/climate.py

```diff
@@ -76,6 +76,10 @@ def has_decimal(self):
         """Return if temperature values support decimal"""
         return self._divider >= 10
 
+    def has_halves(self):
+        """Return if temperature values support halves"""
+        return self._divider % 2 == 0
```

Comment:
Divider can be overridden by home assistant.
If someone wants to divide by 3, he will not have halves.
What the reason behind using this divider to know that the device support halves?